ui-admin/src/study/participants/export/ExportDataControl.tsx

```diff
@@ -67,6 +89,10 @@ const ExportDataControl = ({ studyEnvContext, show, setShow }: {studyEnvContext:
     setOnlyIncludeMostRecent(e.target.value === 'true')
   }
 
+  const inlcudeSubheadersChanged = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setIncludeSubheaders(e.target.value === 'true')
+  }
+
   return <Modal show={show} onHide={() => setShow(false)}>
```

Comment:
I love the advanced options UX as well! I wonder if this modal is becoming hefty enough to justify making the width a big bigger with a `size="xl"`

Reply:
will see what people say during demo